style.css

```diff
@@ -0,0 +1,590 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: lato, sans-serif;
+}
+
+section {
+  padding: 1rem;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.upper {
+  display: none;
+}
+
+header {
+  background-image: url(images/bg1.JPG);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 1rem;
+  padding-bottom: 4rem;
+}
+
+.logo {
+  display: none;
+}
+
+.icon i {
+  font-size: 30px;
+  color: #272a31;
+  z-index: 99;
+}
+
+header nav ul {
+  position: absolute;
+  height: 100vh;
+  width: 100%;
+  background-color: rgba(235, 230, 230, 0.534);
+  top: 0;
+  right: -100%;
+  padding: 2rem 1rem;
+  display: none;
+  transition: 0.5s linear;
+  z-index: 99;
+  backdrop-filter: blur(6px);
+}
+
+header nav li {
+  margin-bottom: 2rem;
+  transition: 0.4s linear;
+}
+
+header nav a {
+  font-size: 23px;
+  color: rgb(58, 58, 58);
+  transition: 0.3s linear;
+}
+
+#tray .icon {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+}
+
+.btn a {
+  display: none;
+}
+
+.program a {
+  color: #d3d3d3;
+  text-transform: uppercase;
+}
+
+nav a:hover {
+  color: #ec5242;
+}
+
+.links {
+  text-transform: capitalize;
+}
+
+footer h3 {
+  border: 2px solid #ec5242;
+  text-align: center;
+  padding: 1px 1rem;
+  font-size: 15px;
+  margin-right: 1rem;
+  color: #272a31;
+}
+
+footer p {
+  font-size: 14px;
+  margin-left: 1rem;
+  color: #272a31;
+}
+
+section h2 {
+  text-align: center;
+  margin: 2rem 0;
+  position: relative;
+}
+
+section h2::after {
+  content: '';
+  width: 35px;
+  height: 2px;
+  border-radius: 12px;
+  position: absolute;
+  right: calc(50% - 8px);
+  bottom: -8px;
+  background-color: #ec5242;
+}
+
+.intro {
+  padding: 2.5rem 0;
+  text-transform: capitalize;
+}
+
+.intro h3 {
+  font-size: 23px;
+  color: #ec5242;
+  font-weight: 400;
+}
+
+.intro h2 {
+  font-size: 28px;
+  margin-top: 15px;
+  font-weight: 800;
+}
+
+.intro h1 {
+  color: #ec5242;
+  font-size: 32px;
+  line-height: 1.1;
+  margin: 0.5rem 0;
+  text-transform: uppercase;
+  font-weight: 900;
+}
+
+.pros p {
+  width: 65%;
+}
+
+.intro p:last-child,
+.intro h2 {
+  color: #272a31;
+}
+
+.bg {
+  background-color: rgb(245 243 243 / 93%);
+  border: 2px solid white;
+  padding: 10px;
+  margin: 1rem 0;
+}
+
+.show {
+  right: 0;
+  display: block;
+}
+
+#larger {
+  display: none;
+}
+
+.program {
+  padding: 2rem 1rem;
+  background-image: url(images/bg3.JPG);
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: #d3d3d3;
+}
+
+.program h2 {
+  text-align: center;
+  text-transform: capitalize;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+}
+
+.pros {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin: 5px 0;
+  padding: 5px;
+  background-color: rgba(255, 255, 255, 0.288);
+  transition: 0.3s linear;
```

Comment:
- [ ] :clap: Good job adding the right background and trying to adjust the cards, while you were doing the changes you forgot to keep the cards on the same line for all screens above 768px of width. please fix it. :recycle: 

- Your design:
![Yprogram](https://user-images.githubusercontent.com/97834160/210816720-0ca1cca4-2f9b-41a2-b7e3-1f262f0030a7.png)
-Behance design:
![cards](https://user-images.githubusercontent.com/97834160/210817004-113f4775-4e67-4122-81cd-375e1b8dc381.png)